diff --git a/src/validation.cpp b/src/validation.cpp
index 004424a784692fc179ee9d1f93a054098e46e9ff..5058665d5daaa745f41fb2b2d6f0fbdb5f004eea 100644
--- a/src/validation.cpp
+++ b/src/validation.cpp
@@ -6681,7 +6681,6 @@ ChainstateManager::ChainstateManager(const util::SignalInterrupt& interrupt, Opt
       m_blockman{interrupt, std::move(blockman_options)},
       m_validation_cache{m_options.script_execution_cache_bytes, m_options.signature_cache_bytes}
 {
-    m_options.notifications.warningSet(kernel::Warning::RULES_NOT_ENFORCED, strprintf(_("Warning: This outdated version does not support the BIP110/RDTS upgrade, and is therefore vulnerable to displaying fake or fraudulent transactions. Please update: %s"), CLIENT_URL));
 }
 
 ChainstateManager::~ChainstateManager()
